x-pack/functionbeat/Makefile

```diff
@@ -8,6 +8,7 @@ GOX_FLAGS=-arch="amd64 386 arm ppc64 ppc64le"
 ES_BEATS?=../../
```

Comment:
For this file, would it be possible to migrate to the more minimal, fully mage based style?

```
#
# Variables
#
ES_BEATS ?= ../..
GOX_FLAGS=-arch="amd64 386 arm ppc64 ppc64le"

#
# Includes
#
include ../../dev-tools/make/xpack.mk
```

Or are there some targets in the Makefile that you must have that are not part of the functionbeat magefile?